src/sqlalchemy_oso_cloud/orm.py

```diff
@@ -14,29 +14,39 @@ class Resource:
   """
   pass
 
+class RoleMapping:
```

Comment:
nit: is it unnecessary to have `Mapping` in this name? It seems redundant or inconsistent. Does "mapping" mean a mapping from ORM models to role assignments (in which case it would seem like `Resource` would have to be named `ResourceMapping` for the naming to be consistent) or does it mean mapping from users to resources via roles? In which case I would think "Assignment" may be more fitting than "Mapping", but still not really necessary. Another idea would be to name it `HasRole`, to make it clearer that it maps to `has_role` facts